### ISSUE TITLE
Add ESLint for linting

### ISSUE THREAD

**Issue by liketurbo** [ton-community/ton-docs#461]
While contributing, I noticed some formatting inconsistency, particularly in indentation style.

It *is* a minor thing, but it does cause ambiguity, which in turn makes it hard to follow [Contributing Code Convention](https://docs.ton.org/contribute/#code-conventions).

Adding [ESLint](https://eslint.org/) with an indentation rule and GitHub linting action to enforce it will mitigate that problem.

**Comment by reveloper** [comment]
@liketurbo Thank you for helping with the issue!
